packages/react-table/src/components/TableComposable/examples/ComposableTable.md

```diff
@@ -351,10 +352,166 @@ ComposableTableSortable = () => {
 };
 ```
 
+### Composable: Sortable with toolbar
```

Comment:
Same comment here about example naming - it can still reference a toolbar since people may like to know that at the top level, too. But I think we should also emphasize the 'responsive' nature of this solution to contrast it to the older 'non-responsive' solution.